### DISCUSSION TITLE
[KCloud Access error][HwX]

### DISCUSSION THREAD

**Discussion by kopter0** [prosyslab-classroom/cs348-information-security#743] [Q&A]
When I try to login I get error: "User is already logged in" (in korean)
How it can be fixed?
Can you force logout user?  my login is cs448-60

(I always use 1 device to log in and since my last login it is been 12 hours)

**Comment by RiceBiscuits** [comment]
Please contact kcloud@kaist.ac.kr regarding KCloud related issues.
